src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  background-color: #e4e4e4;
+
+  --blue-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background-color: white;
+}
+
+.nav {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.list-item {
+  list-style: none;
+  margin: 0 10px;
+
+}
+
+.list-item:first-child {
+  margin: 0;
+}
+
+.list-item:last-child {
+  margin: 0;
+}
+
+.list a {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  text-transform: uppercase;
+}
+
+.list a:hover {
+  text-decoration: underline;
+  text-decoration-color: var(--blue-color);
+  text-decoration-thickness: 2px;
+}
+
+.list .is-active {
+  color: var(--blue-color);
+}
+
+.list .is-active::after {
+  position: absolute;
```

Comment:
The `.is-active::after` pseudo-element uses `position: absolute`, but it may not display correctly unless its parent element is positioned (i.e., has a `position` property other than `static`). Ensure the parent element is positioned to avoid layout issues.